test/pfm/pfm_test.go

```diff
@@ -4,13 +4,14 @@ import (
 	"bytes"
 	"encoding/json"
 	"errors"
+	abci "github.com/cometbft/cometbft/abci/types"
```

Comment:
import ordering looks a little odd :D

Reply:
it was auto added here :D I think we can do a full linting pass at the end anyway so I'm not too worried about things like import ordering just yet.